README.md

```diff
@@ -1,4 +1,5 @@
-## IBM MQ samples and patterns
+## IBM MQ samples and patterns 
+Please Note: the instructions below should NOT be considered for ibm-messagin-mq-cloud-showcase-app
```

Comment:
The repo readme needs an overall. Remove this change from the pull request, as it is misleading and reads "you will spend countless hours learning the basics with the showcase app"